fiasko_bro/code_validator.py

```diff
@@ -237,11 +236,11 @@ def _run_warning_validators_until(self, failed_error_group_name, arguments):
             )
         return warnings
 
-    def validate(self, solution_repo, original_repo=None, **kwargs):
+    def validate(self, path, original_repo=None, **kwargs):
```

Comment:
Let's name it `repo_path`, for the sake of clarity.

Reply:
Ok, but i chose shorter name to get into 79 char limit per line